### DISCUSSION TITLE
Dreambooth folder for a single model is 9.4GB with LORA, should it be so much, can I reduce it somehow?

### DISCUSSION THREAD

**Discussion by a-l-e-x-d-s-9** [d8ahazard/sd_dreambooth_extension#539] [Q&A]
Dreambooth folder for single is 9.4GB for single model with LORA, should it be so much, can I reduce it somehow?
Because LORA should only create a small training, is possible just to use a link to base model, and not copy it?
I'm running out of space... :-(

**Comment by Thomas-MMJ** [comment]
Right now sd_dreambooth is generating checkpoint models during training instead of just training LORAs, you can comment out the checkpoint save if you dig into the code.  The LORAs themselves are just 3 MB.  The checkpoints are at least 2GB each and can be 4GB or more depending on whether you did 16 or 32 bit, etc.

**Comment by a-l-e-x-d-s-9** [comment]
I checked which files are taking most of the space:
- 3.4GB unet/diffusion_pytorch_model.bin  
- 3.4GB unet_lora/diffusion_pytorch_model.bin  
- 1.2GB safety_checker/pytorch_model.bin 
- 0.5GB text_encoder/pytorch_model.bin 
- 0.5GB text_encoder_lora/pytorch_model.bin 

Is the redundancy between regular dreambooth and LORA, in unet and text_encoder, really necessary?
Even if yes, usually people will use only one or the other and not both. Because I'm using LORA, can I just delete unet and text_encoder?
